paasta_tools/autoscaling/autoscaling_service_lib.py

```diff
@@ -77,6 +77,13 @@ class MetricsProviderNoDataError(ValueError):
     pass
 
 
+def average(iterable):
+    """
+    Returns the average value of an iterable
+    """
+    return sum(iterable) / len(iterable)
```

Comment:
wrap this in a try/except and check for ``ZeroDivisionError`` - could happen if you pass this to an empty list.

Also, can you move this to utils.py?